notebooks/06_Serving/seldondeployment.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: machinelearning.seldon.io/v1
+kind: SeldonDeployment
+metadata:
+  name: seldon-model
+spec:
```

Comment:
Does it have canary examples we can add here?

Reply:
Addressed.